lib/src/base-styles/_nx-toast.scss

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../base-styles/nx-alert';
+@use '../scss-shared/nx-container-helpers';
+
+.nx-toast__wrapper {
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
+  @include nx-container-helpers.container-vertical;
+  margin-top: var(--nx-spacing-2x);
+  position: fixed;
+}
+
+.nx-toast {
+  @extend .nx-alert;
+  margin: var(--nx-spacing-2x) 0 0 0;
+  opacity: 0.5;
+  position: relative;
+  width: 500px;
+}
+
+.nx-toast--success {
+  @extend .nx-alert--success;
+}
+
+.nx-toast--warning {
+  @extend .nx-alert--warning;
+}
+
+.nx-toast--error {
+  @extend .nx-alert--error;
+}
+
+.nx-toast--info {
+  @extend .nx-alert--info;
+}
+
+.animate {
```

Comment:
This and the blocks below need to be scoped to just toasts

Reply:
> This and the blocks below need to be scoped to just toasts

Scoped properly here: [96052f2](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/96052f20c90737722ab53cc3a30fa6d67570c22b)